doc/classes/CPUParticles.xml

```diff
@@ -71,134 +95,197 @@
 		<member name="emission_points" type="PoolVector3Array" setter="set_emission_points" getter="get_emission_points">
 		</member>
 		<member name="emission_shape" type="int" setter="set_emission_shape" getter="get_emission_shape" enum="CPUParticles.EmissionShape">
+			Particles will be emitted inside this region. Use [enum EmissionShape] for values. Default value: [constant EMISSION_SHAPE_POINT].
 		</member>
 		<member name="emission_sphere_radius" type="float" setter="set_emission_sphere_radius" getter="get_emission_sphere_radius">
+			The sphere's radius if [enum EmissionShape] is set to [constant EMISSION_SHAPE_SPHERE].
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+			If [code]true[/code], particles are being emitted. Default value: [code]true[/code].
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+			ow rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
 		</member>
 		<member name="flag_align_y" type="bool" setter="set_particle_flag" getter="get_particle_flag">
+			Align y-axis of particle with the direction of its velocity.
 		</member>
 		<member name="flag_disable_z" type="bool" setter="set_particle_flag" getter="get_particle_flag">
+			If [code]true[/code], particles will not move on the z axis. Default value: [code]false[/code].
 		</member>
 		<member name="flag_rotate_y" type="bool" setter="set_particle_flag" getter="get_particle_flag">
+			If [code]true[/code], particles rotate around y-axis by [member angle].
 		</member>
 		<member name="flatness" type="float" setter="set_flatness" getter="get_flatness">
+			Amount of [member spread] in Y/Z plane. A value of [code]1[/code] restricts particles to X/Z plane. Default [code]0[/code].
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+			If [code]true[/code], results in fractional delta calculation which has a smoother particles display effect. Default value: [code]true[/code]
 		</member>
 		<member name="gravity" type="Vector3" setter="set_gravity" getter="get_gravity">
+			Gravity applied to every particle. Default value: [code](0, 98, 0)[/code].
```

Comment:
I think default gravity in 3D is `(0, -9.8, 0)`.

Reply:
ParticlesMaterial had `98` there too. Checked in particles_material.cpp and it's `9.8`